test/unit/parse-siti-payment-file/build-payment-file.test.js

```diff
@@ -95,30 +123,44 @@ describe('SITI payment file batch header is split into batches, headers and line
   })
 
   test('should call validate when valid fileBuffer and sequence are received', async () => {
-    await buildAndTransformParseFile(fileBuffer, sequence)
+    await buildAndTransformParseFile(fileBuffer, schemeType)
     expect(validate).toHaveBeenCalled()
   })
 
   test('should call validate with batchHeaders, batchPaymentRequests and sequence when valid fileBuffer and sequence are received', async () => {
-    await buildAndTransformParseFile(fileBuffer, sequence)
-    expect(validate).toHaveBeenCalledWith(batchHeaders, batchPaymentRequests, sequence)
+    await buildAndTransformParseFile(fileBuffer, schemeType)
+    expect(validate).toHaveBeenCalledWith(batchHeaders, batchPaymentRequestsSFI, schemeType.batchId)
   })
 
   test('should call buildPaymentRequests when valid fileBuffer and sequence are received', async () => {
-    await buildAndTransformParseFile(fileBuffer, sequence)
+    await buildAndTransformParseFile(fileBuffer, schemeType)
     expect(buildPaymentRequests).toHaveBeenCalled()
   })
 
-  test('should call buildPaymentRequests with batchPaymentRequests when validate return true', async () => {
-    await buildAndTransformParseFile(fileBuffer, sequence)
-    expect(buildPaymentRequests).toHaveBeenCalledWith(batchPaymentRequests)
+  test('should call buildPaymentRequests with batchPaymentRequests when validate return true and SFI Pilot input file', async () => {
+    await buildAndTransformParseFile(fileBuffer, schemeType)
+    expect(buildPaymentRequests).toHaveBeenCalledWith(batchPaymentRequestsSFI, 'SFIP')
+  })
+
+  test('should call buildPaymentRequests with batchPaymentRequests when validate return true and SFI input file', async () => {
```

Comment:
Is this testing against `SFI` or `SFI Pilot`?

The test name and change of `schemeType.scheme` to `SFI`  suggests SFI, however the value `SFIP` passed into `buildPaymentRequests` suggests Pilots